meson.build

```diff
@@ -120,8 +119,14 @@ if cc.has_argument('-fvisibility=hidden')
   va_c_args += ['-DHAVE_GNUC_VISIBILITY_ATTRIBUTE']
 endif
 
-if (not WITH_DRM and not WITH_X11 and not WITH_WAYLAND)
-  error('Please install at least one backend dev files (DRM, X11, Wayland)')
+WITH_WIN32 = false
+libwin32_dep = []
+if get_option('with_win32') != 'no'
+    WITH_WIN32 = (host_machine.system() == 'windows')
+endif
```

Comment:
I must be missing something here:
Is this the 3rd patch in the MR - as such one cannot build the project with just the first two commits?

That sounds odd to me - might as well squash this with the first patch in the PR.

Reply:
Got confused there by the UI - this is the second commit, so it's fine as-is. Although it might be worth moving the CI commit to the end of the PR.